### ISSUE TITLE
ModuleNotFoundError: No module named 'openai.openai_object'

### ISSUE THREAD

**Issue by bockaerts** [gkorepanov/whisper-stream#8]
```
    from whisperstream import atranscribe_streaming_simple
  File "C:\Python39\lib\site-packages\whisperstream\__init__.py", line 1, in <module>
    from whisperstream.stream import (
  File "C:\Python39\lib\site-packages\whisperstream\stream.py", line 7, in <module>
    from openai.openai_object import OpenAIObject
ModuleNotFoundError: No module named 'openai.openai_object'
```
I'm getting the following error when running the example code.

**Comment by gkorepanov** [comment]
Use openai==0.28.1
It is very easy to switch to newer openai package versions, I'm planning this in future, but PR is always welcome

**Comment by gkorepanov** [comment]
Fixed by #9
